ci/lint.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DEFAULT_IMAGE=quay.io/helmpack/chart-testing:v3.0.0-rc.1
+
+main() {
+    local image="$DEFAULT_IMAGE"
+    local changed
+    local command=
```

Comment:
extra `=` sign (or missing in the line above)

Reply:
Fixed in 1ed6e8a459984cf0446fe10bdfd4b22cd3c8acae